SCons/Defaults.py

```diff
@@ -480,15 +499,27 @@ def _stripixes(prefix, itms, suffix, stripprefixes, stripsuffixes, env, c=None):
     stripprefixes = list(map(env.subst, flatten(stripprefixes)))
     stripsuffixes = list(map(env.subst, flatten(stripsuffixes)))
 
+    # This is a little funky: if literal_prefix is the same as os.pathsep
+    # (e.g. both ':'), the normal conversion to a PathList will drop the
+    # literal_prefix prefix. Tell it not to split in that case, which *should*
+    # be okay because if we come through here, we're normally processing
+    # library names and won't have strings like "path:secondpath:thirdpath"
+    # which is why PathList() otherwise wants to split strings.
+    do_split = not literal_prefix == os.pathsep
```

Comment:
Does this handle when LIBLITERALPREFIX isn't defined now? Or did you just define this down all the likely paths?

Reply:
paths (in platform). I don't know how to fix otherwise, eval just has to have stuff in one of the two dicts, and don't see another clean way